riptide-failsafe/src/main/java/org/zalando/riptide/failsafe/FailsafePlugin.java

```diff
@@ -0,0 +1,46 @@
+package org.zalando.riptide.failsafe;
+
+import net.jodah.failsafe.CircuitBreaker;
+import net.jodah.failsafe.Failsafe;
+import net.jodah.failsafe.RetryPolicy;
+import org.zalando.riptide.Plugin;
+import org.zalando.riptide.RequestArguments;
+import org.zalando.riptide.RequestExecution;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+public final class FailsafePlugin implements Plugin {
```

Comment:
You are missing the `Fallback` here?

Reply:
`CompletableFuture` has `exceptionally(..)` that already allows you to specify a fallback value. See https://github.com/zalando/faux-pas#completablefutureexceptionally